docs/modules/usage/llms/litellm-proxy.md

```diff
@@ -0,0 +1,55 @@
+
+# LiteLLM Proxy
+
+OpenHands supports using LiteLLM proxy to access various LLM providers. This is particularly useful when you want to:
+
+- Use a single interface to access multiple LLM providers
+- Add authentication, rate limiting, and other features to your LLM API
+- Route requests through a proxy for security or networking requirements
+
+## Configuration
+
+To use LiteLLM proxy with OpenHands, you need to:
+
+1. Set up a LiteLLM proxy server (see [LiteLLM documentation](https://docs.litellm.ai/docs/proxy/quick_start))
+2. Configure OpenHands to use the proxy
+
+Here's an example configuration:
+
+```toml
+[llm]
+# Important: Use `litellm_proxy/` instead of `openai/`
```

Comment:
Just out of curiosity, why is this?

Reply:
This is a weird issue we found in https://github.com/All-Hands-AI/OpenHands/issues/4912 -- not sure why 😓 but it seems to get rid of the bedrock bug

Reply:
Does this mean that the users are supposed to run the litellm proxy? In my understanding, the 'openai/' prefix works regardless whether there's a litellm proxy on the way or not.

Reply:
Ah I see. It might be that litellm proxy needs it now, and I'm not sure if it's possible for our setup to change it, or litellm proxy now really requires it.

On the other hand, the `openai/` prefix should still be usable. I just tried it with other providers, and it works as usual (like openrouter).